.changeset/gold-maps-cut.md

```diff
@@ -0,0 +1,16 @@
+---
+"@fluidframework/aqueduct": minor
+"@fluidframework/container-runtime": minor
+"@fluidframework/container-runtime-definitions": minor
+"@fluidframework/datastore": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/test-runtime-utils": minor
+---
+
+# Removed `_createDataStoreWithProps`
+
+Removed two main APIs, `ContainerRuntime.createDataStoreWithProps` and `IContainerRuntimeBase.createDataStoreWithProps`
+has been removed.
```

Comment:
The `ContainerRuntime.createDataStoreWithProps` and `IContainerRuntimeBase.createDataStoreWithProps` APIs have been removed.